### PR TITLE
Add Run 2.2i dr6c object and metacal catalogs

### PR DESCRIPTION
**Pull request by yymao**
This PR adds Run 2.2i dr6c object and metacal catalogs. 

There are 165 tracts of object catalogs, and 159 tracts for the metacal. The difference (6 tracts) is due to:
- Not available in metacal (4 tracts): 3075, 4034, 4431, 4639
- Mismatched entries (missing patches) (2 tracts): 2907, 4437

This PR also remove "dr6" (which is pointing to "dr6a") to avoid confusion. "dr6a" can still be accessed.

This PR is ready for review but should not be merged until https://github.com/LSSTDESC/desc-help/issues/29 is done and the paths are updated.

### CODE REVIEW

**Comment by yymao** [comment]
File copy has been completed (https://github.com/LSSTDESC/desc-help/issues/29) and the paths are updated. I've also updated the README. This PR has been tested and is now ready for review.

File: GCRCatalogs/catalog_configs/dc2_object_run2.2i_dr6b.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
-based_on: dc2_object_run2.2i_dr6
+subclass_name: dc2_object.DC2ObjectParquetCatalog
 base_dir: ^/DC2-prod/Run2.2i/dpdd/Run2.2i-dr6b/object_table_summary
-description: DC2 Run 2.2i DR6b Object Table
+description: DC2 Run 2.2i DRb Object Table
```
Comment (wmwv) [inline GCRCatalogs/catalog_configs/dc2_object_run2.2i_dr6b.yaml:3]: "DRb" -> "DR6b"

**Comment by yymao** [comment]
Thanks @wmwv. Can you give this a quick re-approval?

**Comment by yymao** [comment]
Actually, I'll merge with my super power since that's a trivial fix.

**Pull request merged**